scripts/workflow-diagnostics/workflow_diagnostics.py

```
@@ -0,0 +1,593 @@
+                    
+                    for event in response.get('events', []):
+                        all_events.append({
+                            'log_group': log_group,
+                            'timestamp': event['timestamp'],
+                            'message': event['message'],
+                            'formatted_time': datetime.fromtimestamp(
+                                event['timestamp'] / 1000
+                            ).strftime('%Y-%m-%d %H:%M:%S')
+                        })
+                        
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)